### PR TITLE
Feature/responsive

### PR DESCRIPTION
**Pull request by CarlosHWolox**
## Summary 
Add responsive behaviour to css styles 

## Trello Card
https://trello.com/c/OsjieA4s/15-dise%C3%B1o-responsive

### CODE REVIEW

File: scss/navbar.scss [hunk 1]
```diff
@@ -19,3 +19,9 @@
   content: url('./assets/wolox_logo.svg');
   min-width: 200px;
 }
+
+@media screen and (max-width: 1024px) {
+  .header-title {
```
Comment (HarryPerez) [inline scss/navbar.scss:24]: good 👍

File: scss/titles.scss [hunk 2]
```diff
@@ -13,3 +13,16 @@
     z-index: -1;
   }
 }
+
+@media screen and (max-width: 1024px){
```
Comment (HarryPerez) [inline scss/titles.scss]: why do you need this media query?. `Front-End and Back-End` titles should not move when the screen width changes.

**Review by HarryPerez — CHANGES_REQUESTED** [review]

**Review by HarryPerez — APPROVED** [review]

**Comment by HarryPerez** [comment]
Excellent job during all your training! 👍 🥇

File: scss/navbar.scss [hunk 3]
```diff
@@ -19,3 +19,9 @@
...
+    visibility: hidden;
```
Comment (SKOLZ) [inline scss/navbar.scss]: what you want here is `display: none;` because display none makes the element disappear and stop taking up space

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**